packages/big-design/src/components/List/Item/Content.tsx

```diff
@@ -84,7 +85,9 @@ export const Content = memo(({ item, isHighlighted }: ContentProps) => {
             <Small color={descriptionColor(disabled)}>{description}</Small>
           </FlexItem>
         ) : (
-          content
+          <StyledLabel content={content} isSelected={isSelected}>
+            {content}
+          </StyledLabel>
```

Comment:
I can see why you're repeating `content` but seems weird to me.

Reply:
I was trying to find better way but for now this only one that works